sdk/servicebus/azure-servicebus/azure/servicebus/management/_management_client.py

```diff
@@ -251,73 +247,610 @@ def update_queue(self, queue_description, **kwargs):
+        topic_description = TopicRuntimeInfo._from_internal_entity(entry.content.topic_description)
+        topic_description.name = topic_name
+        return topic_description
+
+    def create_topic(self, topic, **kwargs):
+        # type: (Union[str, TopicDescription], Any) -> TopicDescription
+        """Create a topic.
+
+        :param Union[str, ~azure.servicebus.management.TopicDescription] topic: The topic name or a `TopicDescription`
+         instance. When it's a str, it will be the name of the created topic. Other properties of the created topic
+         will have default values decided by the ServiceBus.
+         Use a `TopicDescription` if you want to set queue properties other than the queue name.
+        :rtype: ~azure.servicebus.management.TopicDescription
+        """
+        try:
+            topic_name = topic.name  # type: ignore
+            to_create = topic._to_internal_entity()  # type: ignore  # pylint:disable=protected-access
+        except AttributeError:
+            topic_name = topic  # type: ignore
+            to_create = InternalTopicDescription()  # Use an empty topic description.
+
+        create_entity_body = CreateTopicBody(
+            content=CreateTopicBodyContent(
+                topic_description=to_create,  # type: ignore
+            )
+        )
+        request_body = create_entity_body.serialize(is_xml=True)
+        with _handle_response_error():
+            entry_ele = cast(
+                ElementTree,
+                self._impl.entity.put(
+                    topic_name,  # type: ignore
+                    request_body, api_version=constants.API_VERSION, **kwargs)
+            )
+        entry = TopicDescriptionEntry.deserialize(entry_ele)
+        result = TopicDescription._from_internal_entity(entry.content.topic_description)
+        result.name = topic_name
+        return result
+
+    def update_topic(self, topic_description, **kwargs):
+        # type: (TopicDescription, Any) -> None
+        """Update a topic.
+
+        Before calling this method, you should use `get_topic` to get a `TopicDescription` instance, then use the
+        keyword arguments to update the properties you want to update.
+        Only a portion of properties can be updated.
+        Refer to https://docs.microsoft.com/en-us/rest/api/servicebus/update-topic.
+
+        :param ~azure.servicebus.management.TopicDescription topic_description: The topic to be updated.
+        :keyword timedelta default_message_time_to_live: The value you want to update to.
+        :keyword timedelta duplicate_detection_history_time_window: The value you want to update to.
+        :rtype: None
+        """
+
+        if not isinstance(topic_description, TopicDescription):
+            raise TypeError("topic_description must be of type TopicDescription")
+
+        internal_description = topic_description._to_internal_entity()
+        to_update = copy(internal_description)  # pylint:disable=protected-access
```

Comment:
I don't believe I noticed us copying elsewhere.  (My brief recollection is from AIO, but if I'm hallucinating just ignore me)

Is copy sufficient vs. deepcopy, to double check that as well while I"m distracting you with this comment?

Reply:
I removed "copy" because I added deepcopy to TopicDescription._from_internal_entity(). There is no need to copy again any more.